bpf/node_config.h

```diff
@@ -139,6 +139,10 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define CAPTURE6_SIZE 16384
 #endif /* ENABLE_IPV6 */
 
+#ifdef ENABLE_NODEPORT
+#define SNAT_COLLISION_RETRIES 128
+#endif
+
```

Comment:
Looks like this change tripped over a weird issue in BPF program compilation during downgrade:
https://github.com/cilium/cilium/issues/37602